install/app-chrome.sh

```diff
@@ -1,5 +1,7 @@
-cd ~/Downloads
-wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-sudo apt install ./google-chrome-stable_current_amd64.deb
-rm google-chrome-stable_current_amd64.deb
-cd -
+if [[ $ARCH == "amd64" ]]; then
+  wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+  sudo apt install -y ./google-chrome-stable_current_amd64.deb
+  rm google-chrome-stable_current_amd64.deb
+elif [[ $ARCH == "arm64" ]]; then
+  echo "Chrome is not supported on arm64 architecture."
+fi
```

Comment:
Could consider using Chromium instead.

Reply:
we could do that, but in the first place this is DHH's script, and not mine so I won't be calling shots on what to use or what not